### DISCUSSION TITLE
Netbox-docker on AWS ECS

### DISCUSSION THREAD

**Discussion by resparis** [netbox-community/netbox-docker#671] [General]
Hi All, 

Is there a tutorial to use netbox on Amazon Elastic Container Service (ECS)?

Thanks

R

**Comment by kevingoad-arcfield** [comment]
I don't have a tutorial, but it's fairly easy to get going.

We have Netbox behind an ALB w/ RDS (Aurora Serverless V2) and Redis, all deployed via CDK.  We do customize the container, but only to inject SOCIAL_AUTH options into configuration.py at startup.

**Reply by Cortesja** [reply]
hello! 
I need help with this! please if you could look over my discussion and help me! I'm still new to my job, 1st year system engineer so i really have no idea what I'm doing.

https://github.com/netbox-community/netbox-docker/discussions/1605
